plugins/source/gcp/codegen/templates/newapi_list_grpc_mock.go.tpl

```diff
@@ -4,18 +4,19 @@ package {{.Service}}
 
 import (
 	"context"
-	"fmt"
+	"fmt"{{ if not .ChildTable }}
 	"testing"
+	"github.com/cloudquery/plugins/source/gcp/client"{{ end }}
 	"google.golang.org/grpc"
 	"github.com/cloudquery/plugin-sdk/faker"
-	"github.com/cloudquery/plugins/source/gcp/client"
 	{{if .ProtobufImport}}
   pb "{{.ProtobufImport}}"
   {{end}}
 )
 
 
-func create{{.SubService | ToCamel}}(gsrv *grpc.Server) error {
+func create{{.SubService | ToCamel}}(gsrv *grpc.Server) error { {{ range .Relations }}
```

Comment:
This supports mock testing for relations by registering each relation mock server at the parent level, so testing the parent tests all relations